### DISCUSSION TITLE
failed to infer using the onnx file converted from gcn model

### DISCUSSION THREAD

**Discussion by xiongjun19** [pyg-team/pytorch_geometric#3514] [Q&A]
Hi, Dear! 
      I have managed to export a gcn model into an onnx model. but I failed to do inference with it.
      the error message is: 
          ` 
onnxruntime.capi.onnxruntime_pybind11_state.InvalidArgument: [ONNXRuntimeError] : 2 : INVALID_ARGUMENT : Non-zero status code returned while running ScatterElements node. Name:'ScatterElements_116' Status Message: Indices dim=13258 at pos=0 is greater than input dim=2708
`
       following is the steps to reproduce it:

```
git clone https://github.com/xiongjun19/gnn_tutorial.git
   
pip install onnx==1.10.2  onnxruntime==1.9.0
  
cd gnn_tutorial/pyg_tut
 
python first_ex.py
  
python first_export.py gcn_cora.pth gcn_cora.onnx
```

**Comment by rusty1s** [comment]
Thank you. This is super helpful. Will try to have a look ASAP.
